server/src/main/java/org/elasticsearch/action/admin/indices/settings/put/TransportUpdateSettingsAction.java

```diff
@@ -75,9 +87,24 @@ protected ClusterBlockException checkBlock(UpdateSettingsRequest request, Cluste
     protected void masterOperation(Task task, final UpdateSettingsRequest request, final ClusterState state,
                                    final ActionListener<AcknowledgedResponse> listener) {
         final Index[] concreteIndices = indexNameExpressionResolver.concreteIndices(state, request);
+        final Settings requestSettings = request.settings();
+
+
+        final Map<String, List<String>> systemIndexViolations = checkForSystemIndexViolations(concreteIndices, requestSettings);
+        if (systemIndexViolations.isEmpty() == false) {
+            final String message = "Cannot override settings on system indices: "
+                + systemIndexViolations.entrySet()
+                    .stream()
+                    .map(entry -> "[" + entry.getKey() + "] -> " + entry.getValue())
+                    .collect(Collectors.joining(", "));
+            logger.warn(message);
```

Comment:
On backport, we will need to make this a deprecation warning

Reply:
Can I clarify - in the backport, should this raise a deprecation warning but not fail the request?

Reply:
Correct on backport this should result in a deprecation warning and still allow the request to go through.